packages/firebase_remote_config/firebase_remote_config/example/test_driver/firebase_remote_config_e2e.dart

```diff
@@ -22,6 +22,7 @@ void main() {
         'welcome': 'default welcome',
         'hello': 'default hello',
       });
+      await remoteConfig.ensureInitialized();
```

Comment:
Added a call to the previously crashing method to the tests setup - this is why we haven't seen the crash on tests.